custom/icds_reports/tasks.py

```diff
@@ -1861,3 +1862,8 @@ def update_bihar_api_table(target_date):
 def update_child_vaccine_table(target_date):
     current_month = force_to_date(target_date).replace(day=1)
     ChildVaccines.aggregate(current_month)
+
+
+def update_bihar_vaccine_table(target_date):
```

Comment:
it would be great to do it like this. so that calling a single parents function all of them can be aggregated like this
https://github.com/dimagi/commcare-hq/pull/27183/files#diff-b8c5a5e308a203bcaee8ed83e6837edfR1858

Reply:
added